### DISCUSSION TITLE
I2C IMU not working

### DISCUSSION THREAD

**Discussion by MasterLox** [qqqlab/madflight#8] [Q&A] [answered]
Hi, 
I want to use this code with a mpu6050 (i2c) but the code does not compile and I don't know why and how to fix it. 
I does compile when i select any other imu with SPI. But all IMUs with i2c just won't work for me... 

#define IMU_USE   IMU_USE_SPI_BMI270  // IMU_USE_SPI_BMI270, IMU_USE_SPI_MPU9250, IMU_USE_SPI_MPU6500, IMU_USE_SPI_MPU6000, IMU_USE_I2C_MPU9250, IMU_USE_I2C_MPU9150, IMU_USE_I2C_MPU6500, IMU_USE_I2C_MPU6050, IMU_USE_I2C_MPU6000

Can someone help me out? 

Thank you ;D

**Comment by qqqlab** [comment, marked as answer]
Hi @MasterLox,

You didn't post the compiler error message, but I guess you have to add the i2c address of the sensor:

`#define IMU_I2C_ADR  0x68 //IMU I2C address. If unknown, use CLI 'i2c'`

Change 0x68 to the actual address of your sensor.

**Reply by MasterLox** [reply]
yup thank you, this fixed it.
